--- a/devices/ccat/Makefile.am	2019-06-24 08:28:32.000000000 +0200
+++ etherlabmaster_1.5.2+20190904hg33b922p8ea394/devices/ccat/Makefile.am	2019-10-23 14:53:50.068366901 +0200
@@ -46,6 +46,6 @@
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
--- a/devices/cx2100/Makefile.am	2019-06-24 08:28:32.000000000 +0200
+++ etherlabmaster_1.5.2+20190904hg33b922p8ea394/devices/cx2100/Makefile.am	2019-10-23 14:53:50.068366901 +0200
@@ -51,6 +51,6 @@
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
--- a/devices/e1000/Makefile.am	2019-06-24 08:28:32.000000000 +0200
+++ etherlabmaster_1.5.2+20190904hg33b922p8ea394/devices/e1000/Makefile.am	2019-10-23 14:53:50.064366849 +0200
@@ -435,6 +435,6 @@
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
--- a/devices/e1000e/Makefile.am	2019-06-24 08:28:32.000000000 +0200
+++ etherlabmaster_1.5.2+20190904hg33b922p8ea394/devices/e1000e/Makefile.am	2019-10-23 14:53:50.064366849 +0200
@@ -692,6 +692,6 @@
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
--- a/devices/igb/Makefile.am	2019-06-24 08:28:32.000000000 +0200
+++ etherlabmaster_1.5.2+20190904hg33b922p8ea394/devices/igb/Makefile.am	2019-10-23 14:53:50.068366901 +0200
@@ -281,6 +281,6 @@
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
--- a/devices/Makefile.am	2019-06-24 08:28:32.000000000 +0200
+++ etherlabmaster_1.5.2+20190904hg33b922p8ea394/devices/Makefile.am	2019-10-23 14:53:50.064366849 +0200
@@ -230,6 +230,6 @@
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
--- a/examples/dc_rtai/Makefile.am	2019-06-24 08:28:32.000000000 +0200
+++ etherlabmaster_1.5.2+20190904hg33b922p8ea394/examples/dc_rtai/Makefile.am	2019-10-23 14:53:50.064366849 +0200
@@ -44,6 +44,6 @@
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
--- a/examples/mini/Makefile.am	2019-06-24 08:28:32.000000000 +0200
+++ etherlabmaster_1.5.2+20190904hg33b922p8ea394/examples/mini/Makefile.am	2019-10-23 14:53:50.064366849 +0200
@@ -46,6 +46,6 @@
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
--- a/examples/rtai/Makefile.am	2019-06-24 08:28:32.000000000 +0200
+++ etherlabmaster_1.5.2+20190904hg33b922p8ea394/examples/rtai/Makefile.am	2019-10-23 14:53:50.064366849 +0200
@@ -44,6 +44,6 @@
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
--- a/examples/tty/Makefile.am	2019-06-24 08:28:32.000000000 +0200
+++ etherlabmaster_1.5.2+20190904hg33b922p8ea394/examples/tty/Makefile.am	2019-10-23 14:53:50.068366901 +0200
@@ -50,6 +50,6 @@
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
--- a/Makefile.am	2019-06-24 08:28:32.000000000 +0200
+++ etherlabmaster_1.5.2+20190904hg33b922p8ea394/Makefile.am	2019-10-23 14:53:50.068366901 +0200
@@ -93,7 +93,7 @@
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 mydist:
 	hg log --style=changelog $(srcdir) > ChangeLog
--- a/master/Makefile.am	2019-06-24 08:28:32.000000000 +0200
+++ etherlabmaster_1.5.2+20190904hg33b922p8ea394/master/Makefile.am	2019-10-23 14:53:50.068366901 +0200
@@ -95,6 +95,6 @@
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
--- a/tty/Makefile.am	2019-06-24 08:28:32.000000000 +0200
+++ etherlabmaster_1.5.2+20190904hg33b922p8ea394/tty/Makefile.am	2019-10-23 14:53:50.068366901 +0200
@@ -46,6 +46,6 @@
 		INSTALL_MOD_DIR="$(INSTALL_MOD_DIR)" modules_install
 
 clean-local:
-	$(MAKE) -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
+	$(MAKE) -i -C "$(LINUX_SOURCE_DIR)" M="@abs_srcdir@" clean
 
 #------------------------------------------------------------------------------
